dot/rpc/dotUpCodec.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 ChainSafe Systems (ON) Corp.
```

Comment:
bit of a nitpick but maybe rename this file to snake case? or just codec.go?

Reply:
Ok